kustomize/high-availability/ha-postgres.yaml

```diff
@@ -3,8 +3,8 @@ kind: PostgresCluster
 metadata:
   name: hippo-ha
 spec:
-  image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-16.2-0
-  postgresVersion: 16
+  image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-15.7-0
```

Comment:
:question: We want to roll this back?

Reply:
This is the template's doing. I can update the templates to use pg16.